engines/ospray/OSPRayFrameBuffer.cpp

```diff
@@ -54,6 +51,8 @@ void OSPRayFrameBuffer::enableDeflectPixelOp()
     {
         ospCommit(_pixelOp);
         ospSetPixelOp(_frameBuffer, _pixelOp);
+        if (_subsamplingFrameBuffer)
+            ospSetPixelOp(_subsamplingFrameBuffer, _pixelOp);
```

Comment:
I don't know the logic of the pixelop, but it seems to me that Deflect may not work properly after subsampling has been enabled.

Reply:
The pixelop code probably might need an adaption. I have to check that, good point. Not sure if I can test it though with my current setup though.

Reply:
The 'normal' deflect streaming works, but the pixelop image is too small.